### PR TITLE
Add non-Zeebe Community Clients to docs

### LINKED ISSUE #2068: Add non-Zeebe Community Clients to docs
In APIs & Tools, we list our Zeebe clients, but we have a few non-Zeebe clients maintained by our community that should also be included for better visibility. 

Increasing the visibility will help us determine if we should move to support these projects fully.

The following clients should be linked as community-supported clients:

* https://github.com/camunda-community-hub/console-client-node-js
* https://github.com/camunda-community-hub/operate-client-node-js
* https://github.com/camunda-community-hub/camunda-operate-client-java
* https://github.com/camunda-community-hub/dotnet-custom-tasklist
* https://github.com/camunda-community-hub/camunda-tasklist-client-java
* https://github.com/camunda-community-hub/tasklist-client-node-js
* https://github.com/camunda-community-hub/optimize-client-node-js

### PR DESCRIPTION
**Pull request by akeller**
## Description

Closes #2068. 

~Changes are only in `/next/`.~ Looking for feedback on tab format on overview page and new sidebar nest (no impact to URLs) of Zeebe clients.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]
Really happy with this 👍

File: docs/apis-tools/community-clients/index.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+[
+{label: 'Zeebe', value: 'zeebe' },
+{label: 'Tasklist', value: 'tasklist' },
+{label: 'Operate', value: 'operate' },
+{label: 'Other', value: 'other' }
+]}>
+
+<TabItem value='zeebe'>
+
+In addition to the core Java and Go clients, there are a number of community-maintained Zeebe client libraries:
```
Comment (pepopowitz) [inline docs/apis-tools/community-clients/index.md:24]: Having text in one tab item but not the rest creates some visual bouncing. What do you think about moving this sentence above the tabs, and rewording to something more general? e.g. 

> In addition to the core Camunda-maintained clients, there are a number of community-maintained component libraries:

This is a non-blocking suggestion, but if it resonates and you want to make these changes, great.

**Review by pepopowitz — APPROVED** [review]
I left one note for possible changes, but it's non-blocking.

Also, for ease of reviewing, here's a video of the tabs in action:

https://github.com/camunda/camunda-platform-docs/assets/1627089/af75a732-5e0a-4496-a939-35f354e98055

File: docs/apis-tools/community-clients/index.md (hunk 1 above)
Comment (akeller) [reply docs/apis-tools/community-clients/index.md]: Great idea! I added it.

**Pull request merged**